### ISSUE TITLE
Print an account of test statuses 

### ISSUE THREAD

**Issue by topepo** [r-lib/rcmdcheck#106]
When running `R CMD check` it would be good to know how many tests were skipped even if there are no failures.

**Comment by jennybc** [comment]
Specifically, I'd rejoice if this:

```
* checking tests ...
  Running ‘testthat.R’
 OK
```

got an upgrade to include more of this:
```
══ Results ══════════════════════
Duration: 14.7 s

OK:       390
Failed:   0
Skipped:  3
```

**Comment by gaborcsardi** [comment]
OK, so this is actually a bit more involved, because parsing this information from the test results is not reliable, it depends on the reporter you are using.

Ideally, we would get the results directly from testthat, as an R object: https://github.com/r-lib/testthat/issues/875

Until that is done, I'll have to postpone this.

**Comment by hadley** [comment]
Can we easily add extra data here? Maybe this should be done more from the testthat side? (i.e. what could `test_check()` emit to display extra info here?)

**Comment by gaborcsardi** [comment]
I think that with the printing of the test results on GHA (and optionally locally as well), this is not so important any more. Locally you can always look at the `.Rout` file if you are interested. No?

**Comment by gaborcsardi** [comment]
OTOH, we could still add some reporting to testthat, I think that would open up some new possibilties. E.g. testthat could write a summary of the results to a file in some format, if an env var is set?

**Comment by hadley** [comment]
I think this is really just a duplicate of #121

**Comment by gaborcsardi** [comment]
Yeah, I closed that one, this one is older and has more participants.

**Comment by hadley** [comment]
I think this is still important; it'd be really nice to be able to see the number of skipped tests without having to dig down further.
